src/backend/distributed/executor/multi_utility.c

```diff
@@ -2974,8 +3043,12 @@ PostProcessUtility(Node *parsetree)
 	indexForm = (Form_pg_index) GETSTRUCT(indexTuple);
 	indexForm->indisvalid = true;
 
+#if (PG_VERSION_NUM >= 100000)
+	CatalogTupleUpdate(pg_index, &indexTuple->t_self, indexTuple);
+#else
 	simple_heap_update(pg_index, &indexTuple->t_self, indexTuple);
 	CatalogUpdateIndexes(pg_index, indexTuple);
+#endif
```

Comment:
I still don't think it's good to do this, and that we should just add our own CatalogTupleUpdate() etc wrappers for the older releases.  I'm pretty sure I pointed this out before.